### PR TITLE
Create application announcement by resume and add key search

### PR DESCRIPTION
**Pull request by Jirattikarn**
- Add image to use on the application page 
- import MainLayout to use in the application 
- Connect to API of announcement resume 
- Add getResumeByUserId in application context 
- Edit style of the application page 

Related card: [🧡 [SP5] Create Application Announcement by resume](https://trello.com/c/bO6VLi0R)

image

- Add more key search of announcement info
Related card: [🧡 [SP2] Add key search for announcement info management](https://trello.com/c/hHBbtO48)

### CODE REVIEW

**Comment by Jirattikarn** [comment]
[🧡 [SP5] Create Application Announcement by resume](https://trello.com/c/bO6VLi0R/165-%F0%9F%A7%A1-sp5-create-application-announcement-by-resume)

**Comment by Jirattikarn** [comment]
[🧡 [SP2] Add key search for announcement info management](https://trello.com/c/hHBbtO48/130-%F0%9F%A7%A1-sp2-add-key-search-for-announcement-info-management)

File: features/Academic-Industry/services/apiAcademicIndustry.tsx [hunk 1]
```diff
@@ -88,46 +88,43 @@ const apiAcademic = {
...
+  },
+  getAllAnnouncementResumes: async () => {
+    const response = await api.get('/academic-industry/applications')
+    return response
+  },
+  getAnnouncementResumeByAdmin: async () => {
+    const response = await api.get(`academic-industry/admin/applications `)
+    return response
+  },
+  CreateAnnouncementResume: async (data) => {
```
Comment (mild-tn) [inline features/Academic-Industry/services/apiAcademicIndustry.tsx:111]:
```suggestion
  createAnnouncementResume: async (data) => {
```

File: features/Academic-Industry/context/announcement_application_page_context.tsx [hunk 2]
```diff
@@ -29,7 +42,7 @@ export class AnnouncementApplicationFormContext {
 
   createApplication = async (data) => {
     try {
-      await apiAcademic.createApplication(data).then(() => {
+      await apiAcademic.CreateAnnouncementResume(data).then(() => {
```
Comment (mild-tn) [inline features/Academic-Industry/context/announcement_application_page_context.tsx:45]:
```suggestion
      await apiAcademic.createAnnouncementResume(data).then(() => {
```

**Review by mild-tn — APPROVED** [review]

**Pull request merged**